styles.css

```diff
@@ -1,3 +1,83 @@
+* {
+  box-sizing: border-box;
+}
+
+.main {
+  max-width: 600px; /* be 100% width, until 600px */
+  margin: 0 auto; /* 0 top and bottom margin, auto left and right margin */
```

Comment:
if these comments are for your own learning, good. If they are for clarification purposes for other readers of your code, the comments are redundant.